### DISCUSSION TITLE
release notes

### DISCUSSION THREAD

**Discussion by hape42** [hape42/dailygammon#18] [Ideas]
Current (as of November 19, 2022) version is 1.42

I would suggest to first bring a smaller next version 1.43 into the Appstore.
What do you think?

We also need notes

I can think of so far:

- Faster loading of the TopPage
- ?

**Comment by kagsteiner** [comment]
If it's not too much effort for you, as a user I always found it good to get small updates that change this or that.

I have nothing of user value to contribute :-)

**Comment by hape42** [comment]
done
